WooCommerce/Classes/ViewRelated/Bookings/Booking Details/BookingDetailsView.swift

```diff
@@ -75,58 +85,262 @@ private extension BookingDetailsView {
+            HStack {
+                Text(headerContent.serviceName)
+                Circle()
+                    .fill(.tertiary)
+                    .frame(
+                        width: Layout.circleSeparatorSize,
+                        height: Layout.circleSeparatorSize
+                    )
+                Text(headerContent.customerName)
+            }
+            .font(.footnote.weight(.medium))
+            .foregroundColor(.secondary)
+
             HStack {
                 ForEach(headerContent.status, id: \.self) { status in
                     Text(status.labelText)
-                        .font(.caption)
-                        .padding(4)
+                        .font(.caption2)
+                        .padding(.vertical, 4.5)
+                        .padding(.horizontal, 8)
                         .background(status.labelColor)
                         .cornerRadius(4)
                 }
             }
             .padding(.top, Layout.headerBadgesAdditionalTopPadding)
         }
         .frame(maxWidth: .infinity, alignment: .leading)
-        .padding(.vertical, 6)
+        .padding(.vertical)
+    }
+
+    func attendanceView(with content: BookingDetailsViewModel.AttendanceContent) -> some View {
+        TitleAndValueRow(
+            title: Localization.statusRowTitle,
+            value: .placeholder(content.value),
+            selectionStyle: .disclosure,
+            horizontalPadding: 0
+        )
     }
 
     func appointmentDetailsView(with content: BookingDetailsViewModel.AppointmentDetailsContent)  -> some View {
         VStack(alignment: .leading, spacing: 0) {
             ForEach(content.rows) { row in
-                TitleAndTextFieldRow(
+                TitleAndValueRow(
                     title: row.title,
-                    placeholder: String(),
-                    text: .constant(row.value),
-                    fieldAlignment: .trailing,
-                    keyboardType: .default,
-                    titleFont: BookingDetailsView.TextFont.bodyMedium,
-                    valueColor: .secondary,
-                    valueFont: BookingDetailsView.TextFont.bodyRegular,
-                    horizontalPadding: 0 // Parent section padding is added elsewhere,
+                    value: .placeholder(row.value),
+                    horizontalPadding: 0,
+                    isMultiline: false
                 )
 
-                if row.id != content.rows.last?.id {
-                    Divider()
-                        .padding(.trailing, -Layout.contentSidePadding)
+                Divider()
+                    .padding(.trailing, -Layout.contentSidePadding)
+            }
+
+            Button {
+                /// On cancel booking button tap
+            } label: {
+                Text(Localization.cancelBooking)
+            }
+            .buttonStyle(SecondaryButtonStyle())
+            .padding(.vertical, Layout.contentVerticalPadding)
```

Comment:
I don't see this button in i2 design, it seems to have been moved to the quick action sheet. Could you please confirm with the designer?

<img width="291" height="542" alt="image" src="https://github.com/user-attachments/assets/9d91fce7-9070-4e72-9925-c78386e70345" />